docs/cli.md

```diff
@@ -293,6 +293,7 @@ You also can specify a constraint when adding a package, like so:
 ```bash
 poetry add pendulum@^2.0.5
 poetry add "pendulum>=2.0.5"
+poetry add "pendulum==2.0.5"
```

Comment:
Not sure if this belongs here. Those two existing entries are the same, but the one you added has different behavior. I would either move it to a separate block or remove it entirely.

Reply:
They are not exactly the same, since `^2.0.5` will pick up `>=2.0.5,<3.0.0` while `>=2.0.5` will not have an upper bound and could pick up `3.0.0` for instance.

I've tried something in https://github.com/python-poetry/poetry/pull/5874/commits/51c975daae2f6b69c0e11b60f2ee1803d9586044 to better detail the possibilities, let me know what you think.